docs/docs/06-migration/v28.0.0.md

```diff
@@ -0,0 +1,124 @@
+
+## **Ignite and Cosmos SDK Upgrade Guide: From Ignite v0.27.0 to v28.0.0 and Cosmos SDK v0.47 to v0.50**
+
+### **Introduction**
+
+This guide provides a step-by-step process for developers to upgrade their applications from Ignite version 0.27.0 to 28.0.0, along with an upgrade in the Cosmos SDK from version 0.47 to v0.50. It covers essential changes, new features, and adjustments required for a smooth transition.
+
+### **Prerequisites**
+
+- Backup your current project.
+- Ensure you have Ignite v0.27.0 and Cosmos SDK v0.47 installed.
+- Basic familiarity with command line operations and the existing project structure.
+
+### **Step 1: Update Ignite CLI to Version 28.0.0**
+
+- **Command**: Run **`curl https://get.ignite.com/cli@v28.0.0 | bash`** in your terminal.
+- **Note**: This command updates the Ignite CLI to the latest version. Ensure you have the necessary permissions to execute it.
+
+### **Step 2: Update Scaffold Chain Command**
+
+- **Old Command**: **`ignite scaffold chain github.com/alice/blog`**
+- **New Command**: **`ignite scaffold chain blog`**
+- **Explanation**: The command format has been simplified in the new version for ease of use.
+
+### **Step 3: Docker Version Upgrades**
+
+- **Action**: Upgrade the Ignite version for the Docker container to match the CLI version.
+- **Note**: Ensure Docker compatibility with the new Ignite CLI version.
+
+### **Step 4: Change in Module Path**
+
+- **Old Path**: **`x/blog/module.go`**
+- **New Path**: **`x/blog/module/module.go`**
+- **Explanation**: The module path structure has been updated for better organization.
+
+### **Step 5: Frontend Scaffolding Options**
+
+- **Action**: Choose between Vue, React, Go, or TypeScript for frontend scaffolding.
+- **Commands**:
+    - **`ignite scaffold react`**
+    - **`ignite scaffold vue`**
+- **Note**: Vue is no longer the default option for frontend scaffolding.
+
+### **Step 6: Update Scaffold Message for CreatePost Command**
+
+- **Action**: Review and update the output for the scaffolded createPost command as per the new format.
+
+### **Step 7: AutoCLI Path Change**
+
+- **Old Path**: **`x/blog/client/cli/tx_create_post.go`**
+- **New Path**: **`x/blog/module/autocli.go`**
+- **Explanation**: AutoCLI is now integrated at a different path to streamline command-line interactions.
+
+### **Step 8: Adjustment in Stored Game**
+
+- **Old Code**:
+    
+    ```go
+    store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.PostKey))
+    
+    ```
+    
+- **New Code**:
+    
+    ```go
+    storeAdapter := runtime.KVStoreAdapter(k.storeService.OpenKVStore(ctx))
+    store := prefix.NewStore(storeAdapter, types.KeyPrefix(types.PostKey))
+    
+    ```
+    
+- **Explanation**: The way the KVStore is accessed has changed, requiring an update in the code for stored games.
+
+### **Step 9: Chain-ID Requirements in CLI Transaction Commands**
```

Comment:
Have we now verified it was really required even if it is in the client.toml? It should be not be required afaik.

Reply:
Yes, I do get the error message `chain ID required but not specified`